### PR TITLE
docs: draft RFC-0360: NFT sale via Mimblewimble invoice

### PR DESCRIPTION
**Pull request by philipr-za**
Description
---
This PR provides a draft RFC for the process to negotiate an NFT sale as an Invoice. This is where the initiator of the transaction requests the payment amount from the counterparty.

Motivation and Context
---

How Has This Been Tested?
---

### CODE REVIEW

File: RFC/src/RFC-0360_NFTInvoices.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+or allow for one party to change the elements provided by the counterparty without making the transaction invalid.
+
+## Related Requests for Comment
+
+* [RFC-0201: TariScript](RFC-0201_TariScript.md)
+* [RFC-0300: Digital Assets Network](RFC-300_DAN.md)
+
+## Description
+The standard interactive [Mimblewimble] transaction is usually initiated by a party, the Sender, that is going to send 
+funds to a Receiver. The Sender selects a number of input UTXO's from which the sent funds and fees will come from, they
```
Comment (delta1) [inline RFC/src/RFC-0360_NFTInvoices.md]:
```suggestion
funds to a Receiver. The Sender selects a number of input UTXOs from which the sent funds and fees will come from, they
```

File: RFC/src/RFC-0360_NFTInvoices.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+build a change UTXO to receive any remaining funds and caluculate the excess of these inputs and change output. The
```
Comment (delta1) [inline RFC/src/RFC-0360_NFTInvoices.md]:
```suggestion
build a change UTXO to receive any remaining funds and calculate the excess of these inputs and change output. The 
```

File: RFC/src/RFC-0360_NFTInvoices.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+* [RFC-0201: TariScript](RFC-0201_TariScript.md)
+* [RFC-0300: Digital Assets Network](RFC-300_DAN.md)
+
+## Description
+The standard interactive [Mimblewimble] transaction is usually initiated by a party, the Sender, that is going to send 
+funds to a Receiver. The Sender selects a number of input UTXO's from which the sent funds and fees will come from, they
+build a change UTXO to receive any remaining funds and caluculate the excess of these inputs and change output. The 
+Receiver is then told how many Tari they will receive and provides the aggregated public excess and public nonce of the 
+Sender's portion of the transaction signature. The Receiver builds a commitment with the specified value and their 
+private spending key and then provide their partial signature for their UTXO and the excess. This gets sent back to the
```
Comment (delta1) [inline RFC/src/RFC-0360_NFTInvoices.md]:
```suggestion
private spending key and then provides their partial signature for their UTXO and the excess. This gets sent back to the 
```

File: RFC/src/RFC-0360_NFTInvoices.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+In the following equations the capital letter subscripts, _S_ and _B_ refer to a token _Seller_ and _Buyer_ respectively.
+
+### Negotiation
+The Seller initiates the negotiation by collecting and calculating the following:
+
+| Terms                                         | Description                                                                                               |
+|--------------------------------------         |-----------------------------------------------------------------------------------------------------------|
+| \\( C_{St} = 0 \cdot H  + k_{St} \cdot G \\)  | Select the Seller's token UTXO to be offered                                                              |
+| \\( C_{Sp} = v_p \cdot H + k_{Sp} \cdot G \\) | Choose a value to be requested from Buyer and select a spending key for the payment UTXO to be received   |
+| \\( R_S = r_S \cdot G) \\)                    | Choose a nonce                                                                                            |
```
Comment (delta1) [inline RFC/src/RFC-0360_NFTInvoices.md]: I think there's a stray bracket here 

```suggestion
| \\( R_S = r_S \cdot G \\)                    | Choose a nonce                                                                                            |
```

**Review by delta1 — COMMENTED** [review]
Looks good so far, just a few minor typos

**Review by stringhandler — APPROVED** [review]
Looks good. There may be some details that crop up during implementation, but this seems solid to me

**Pull request merged**